log.c

```diff
@@ -266,6 +295,23 @@ int pv_log_start(struct pantavisor *pv, const char *rev)
 	return 0;
 }
 
+void __log_to_console(char *module, int level, const char *fmt, ...)
```

Comment:
The printing stuff in log.h is for when logsink is not up  yet. I would keep it this way, so that we can use pv_log all the time, even when logsink is not up.

I like the idea of hiding logsink behind log.c almost completely. "Almost" because pvlogger still needs direct access to logsink. To do that, we need to add functions in log.c that call `pv_logsink_toggle` and `pv_logsink_stop`. So kinda a function in log.c that handles revision transitioning. We could also put the ph_logger transitioning functionality in there. Any thoughts about that?